mythx_cli/cli.py

```diff
@@ -189,7 +194,9 @@ def cli(
         ctx.obj["client"] = Client(
             username=username, password=password, middlewares=[toolname_mw]
         )
-    else:
+    # elif sys.argv[1] not in ("fuzz",):
```

Comment:
I guess we can remove the commented line as it does nothing.

Reply:
done